lang/en/texts/google-summer-of-code.html

```diff
@@ -21,6 +21,50 @@ <h2>Areas where your help can make a big difference</h2>
+    <li>Information on how you would approach the project, what time you can put into it and what you think you will be able to achieve over the GSoC period</li>
+    </ul>
+
+    
+<h3>Make a Small Contribution</h3>
+<p>We want to make sure that new contributors are familiar with our development workflow and with the tools involved with it. Make at least one small code contribution to an Open Food Facts project (preferribly related to the project you want to work on during GSoC) to demonstrate you can build the application from the source code, make changes to it, and submit a merge request in the project's GitHub repository.</p>
+    
+<h3>Decide on a GSoC project</h3>
+<p>Our mentors put together a list of project ideas that you can choose. Project proposals are not strictly limited to the ones listed in this project ideas page. You can contact the main developers or any Open Food Facts project on a public channel, introduce yourself, and suggest your own project ideas. Base any ideas you propose on a small research about the needs of a projects and/or users and make sure there is interest from a mentor before basing your application on it.</p>
+
+    <p>To learn how to contribute code to Open Food Facts, go through our <a href="">Newcomers tutorial</a>.</p>
+    
+    
+<h3>Fill out the Application</h3>
+
+    
+<p>Once the application period has opened, you have to submit your application on the Google Summer of Code website. Your application must be written in English. It should contain a detailed description of your project proposal.</p>
+
+<p>Copy our <a href="https://docs.google.com/document/d/1Xb6aOlPxlnvsGLoyROyGk-qnOjFozQ7jqupfbV8H2u0/edit?usp=sharing">Google Document template</a> and make sure you answer all of the questions.</p>
+
+<p>Please be factual and clear in answering these questions. Feel free to add anything else that is relevant for your application. It is never too early to start working on your GSoC application! Note that GSoC positions are very competitive (with about 4 applicants for one position in the past) The key to creating a strong proposal is to propose a manageable and agreed-upon project, make a contribution to the module your proposal is related to, and write an application that clearly demonstrates your knowledge, skills, and enthusiasm.</p>
+    
+    
+The following things can help you select a project and prepare your idea:
+
+<ul>
+    <li><a href="https://slack.openfoodfacts.org/">Join our chat room</a></li>
+    <li>Introduce yourself in the #summerofcode channel</li>
+    <li>Join the channels related to the language/project/countries you're interested in</li>
+    <li>It is important to first familiarize with the software. Try to make it run on your device/machine.</li>
+    <li>Once you're able to run it, try one of our "good first issues" on GitHub</li>
+    <li>Please, go through the available tutorials and direct your questions to the Slack chat, in the most specific channel possible.</li>
```

Comment:
```suggestion
    <li>Please, go through the available tutorials and direct your questions to the Slack chat, in the most specific channel possible.</li>
    ```suggestion
    <li>As you learn, it is also a good idea to propose updates to the documentation.</li>
```
```